Backend/CarRentalApp/CarRentalApp/Models/DTOs/Requests/UserRegistrationDTO.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using CarRentalApp.Models.ValidationAttributes;
+
+namespace CarRentalApp.Models.DTOs.Requests
+{
+    public class UserRegistrationDTO
+    {
+        [Required]
+        [Email(ErrorMessage = "Incorrect format: The {0} is invalid")]
+        public string Email { get; set; }
+
+        [Required]
+        [StringLength(
+            25,
+            ErrorMessage = "Incorrect format: " +
+                           "The {0} value cannot exceed {1} characters"
+        )]
+        public string Username { get; set; }
+
+        [Required]
+        [RegularExpression(
+            "(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z]).{5,25}",
+            ErrorMessage = "Incorrect format: The {0} value must have " +
+                           "length between 5 and 25 characters and contain at least " +
+                           "one digit, one lowercase letter, one uppercase letter"
+        )]
```

Comment:
```suggestion
        [RegularExpression(
            "(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z]).{5,25}",
            ErrorMessage = "Incorrect format: The {0} value must have length between 5 and 25 characters and contain "
                + "at least one digit, one lowercase letter, one uppercase letter"
        )]
```

it is recommended to keep length of the line equal 120